### PR TITLE
added service category and user react files

### PR DESCRIPTION
**Pull request by texzone**

### CODE REVIEW

**Review by BuhQuick — APPROVED** [review]
This all LGTM, although I think your story for this sprint was just to do the User page. You should create and/or pull in the related stories for your work done here. 

Approving the PR, but we shouldn't merge until the comments in https://github.com/triv202/cs4500_sp19_random1/pull/16 are addressed.

NOTE:
I've noticed that for this PR and the corresponding PR above you're merging from a local fork. It would be helpful to me if you start doing work on branches from the main repository. That way, if I want to pull down your branch and run your changes I can.

**Review by BuhQuick — CHANGES_REQUESTED** [review]
A change was made since I approved this PR --
Please update your Services to use the new dynamic URLs
(see
https://github.com/BuhQuick/cs4500_sp19_client_react_random1/pull/8/files
)

**Review by texzone — COMMENTED** [review]
I've changed all the services to use the middle tier url config

File: src/services/UserService.js [hunk 1]
```diff
@@ -7,9 +8,9 @@ export default class UserService {
         return this.instance
     }
     findUserById = userId =>
-        fetch(`http://localhost:8080/api/users/${userId}`)
+        fetch(MIDDLE_TIER_URL + "/api/users/`" + userId)
```
Comment (BuhQuick) [inline src/services/UserService.js]: There's a ` in the /api/users/ string

File: src/services/UserService.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        if(UserService.instance === null) {
+            UserService.instance = new UserService()
+        }
+        return this.instance
+    }
+    findUserById = userId =>
+        fetch(MIDDLE_TIER_URL + "/api/users/`" + userId)
+            .then(response => response.json())
+    findAllUsers = () =>
+        fetch(MIDDLE_TIER_URL + "api/users")
```
Comment (BuhQuick) [inline src/services/UserService.js]: Need the / before api

**Review by BuhQuick — CHANGES_REQUESTED** [review]
Just a few typos in the userservice  class, otherwise lgtm

File: src/services/UserService.js (hunk 1 above)
Comment (texzone) [reply src/services/UserService.js]: Good catch

**Review by BuhQuick — APPROVED** [review]

**Pull request merged**